util/prep-bazel-airgapped-build.sh

```diff
@@ -154,9 +154,9 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
     @rust_linux_aarch64_toolchains//... \
     @rust_linux_x86_64_toolchains//... \
     @rust_windows_x86_64_toolchains//...
-  cp -R $(${BAZELISK} info output_base)/external/${BAZEL_PYTHON_WHEEL_REPO} \
+  cp -R "$(${BAZELISK} info output_base)"/external/${BAZEL_PYTHON_WHEEL_REPO} \
```

Comment:
Does `${BAZEL_PYTHON_WHEEL_REPO}` not need quoting?

Reply:
Not according to shellcheck since this is a variable expansion rather than command substitution.

Reply:
I guess shellcheck is confident that the variable doesn't contain spaces?

```console
% cat /tmp/script.sh
#!/bin/bash
my_var="foobar"
echo "hello"${my_var}
%
% shellcheck /tmp/script.sh
%
% cat /tmp/script.sh
#!/bin/bash
my_var="foo bar"
echo "hello"${my_var}
%
% shellcheck /tmp/script.sh

In /tmp/script.sh line 3:
echo "hello"${my_var}
            ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
echo "hello""${my_var}"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

Reply:
Actually, I think the reason it's not complaining is that the entire line is already quoted, so there's no need to quote this expansion in particular. See [SC2086](https://www.shellcheck.net/wiki/SC2086).

Reply:
Oh shellcheck does flag it but just as an info-level message rather than a warning.

Reply:
Gotcha. I'm fine with leaving it if it's explicitly below the threshold we care about.